### PR TITLE
Update squash.rb

### PR DESCRIPTION
**Pull request by talum**

### CODE REVIEW

**Review by tracylum — APPROVED** [review]

File: squash.rb [hunk 1]
```diff
@@ -1 +1,3 @@
 squash.rb
```
Comment (talum) [inline squash.rb:1]: test

Comment (talum) [inline squash.rb:1]: another test

File: squash.rb [hunk 2]
```diff
@@ -1 +1,3 @@
...
+
+gourds!
```
Comment (talum) [inline squash.rb:3]:
```suggestion
bleh?
```

Comment (talum) [reply squash.rb:3]: ?

**Comment by talum** [comment]
?

File: squash.rb (hunk 2 above)
Comment (talum) [reply squash.rb:3]: replying to existing thread

File: squash.rb (hunk 1 above)
Comment (talum) [inline squash.rb:1]: making a new comment on R1

**Comment by talum** [comment]
testing

File: squash.rb (hunk 1 above)
Comment (talum) [reply squash.rb:1]: no suggestion application options here

Comment (talum) [reply squash.rb:1]:
```suggestion
hi
```

Comment (talum) [reply squash.rb:1]:
```suggestion
squash.rb
```

Comment (talum) [reply squash.rb:1]:
```suggestion
squash.rb change me
```

**Review by tracylum — CHANGES_REQUESTED** [review]
sdf

**Pull request closed without merging**